apps/ledger-live-mobile/src/screens/Swap/Form/ErrorBanner.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useCallback, useMemo } from "react";
+import { Linking } from "react-native";
+import { useTranslation } from "react-i18next";
+import { getProviderName } from "@ledgerhq/live-common/exchange/swap/utils/index";
+import { ValidCheckQuoteErrorCodes } from "@ledgerhq/live-common/exchange/swap/types";
+import { urls } from "../../../config/urls";
+import { Banner } from "./Banner";
+
+export function ErrorBanner({
+  provider,
+  errorCode,
+}: {
+  provider: string;
+  errorCode: ValidCheckQuoteErrorCodes;
+}) {
+  const { t } = useTranslation();
+
+  const onPress = useCallback(() => {
+    // @ts-expect-error
+    Linking.openURL(urls.swap.providers[provider]?.support);
```

Comment:
shouldn't we just check `urls.swap.providers[provider]` before calling `openURL`, this would avoid the use of `// @ts-expect-error`